### ISSUE TITLE
Localizable asset alt text

### ISSUE THREAD

**Issue by ranuss** [statamic/ideas#624]
Asset alt texts should be localizable as well. This is missing if i get it right.

**Comment by jasonvarga** [comment]
Duplicate of #223
